File: Assignment13/Program3.c
#include<stdio.h>

void print_odd_numbers(int limit)
{ 
    int iCnt = 0;
    iCnt = 1;
  while(iCnt <= limit )
  {
    if(iCnt % 2 != 0)
    {
    printf("%d\t",iCnt);
    }
    iCnt++;
  }
}
int main()
{
    int limit;

    printf("Enter number: ");
    scanf("%d",&limit);

    print_odd_numbers(limit);
    return 0;
}

//Time Complexity = 0(N)

File: Assignment18/Program3.java
class Logic
{
    void printOddNumbers(int n)
    {
        int iCnt = 0;

        for(iCnt = 1;iCnt <= n; iCnt++)
        {
          if(iCnt % 2 == 1)
        {
            System.out.println(iCnt);
        }
        }
        
    }
}
class Program3
{
    public static void main(String A[])
    {
        Logic obj = new Logic();
        obj.printOddNumbers(20);
    }
}